scripts/fdbshow

```diff
@@ -130,7 +130,7 @@ class FdbShow(object):
                     except Exception:
                         vlan_id = bvid
                         print("Failed to get Vlan id for bvid {}\n".format(bvid))
-            self.bridge_mac_list.append((int(vlan_id),) + (fdb["mac"],) + (if_name,) + (fdb_type,))
+            self.bridge_mac_list.append((int(vlan_id or 1),) + (fdb["mac"],) + (if_name,) + (fdb_type,))
```

Comment:
Please check for 'None' and skip those entries. We don't expect FDB to be learnt on Vlan 1.

Reply:
Gotcha - I saw host MACs and mistaking assumed traffic was being forwarded on default VLAN.